etc/kayobe/kolla/globals.yml

```diff
@@ -10,12 +10,19 @@ enable_docker_repo: "{% raw %}{{ 'overcloud' not in group_names or ansible_facts
 # This is necessary for os migrations where mixed clouds might be deployed
 kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}"
 
-kayobe_image_tags:
-  openstack:
-    rocky: zed-rocky-9-20230821T155947
-    ubuntu: zed-ubuntu-jammy-20230821T155947
-
-openstack_tag: "{% raw %}{{ kayobe_image_tags['openstack'][kolla_base_distro] }}{% endraw %}"
+# Dict of Kolla image tags to deploy for each service.
+# Each key is the tag variable prefix name, and the value is another dict,
+# where the key is the OS distro and the value is the tag to deploy.
+# NOTE: This is defined in etc/kayobe/kolla-image-tags.yml.
+kolla_image_tags: "{{ kolla_image_tags | to_nice_yaml(indent=2) }}"
```

Comment:
How can this reference itself?

Reply:
Yeah it's a bit weird, but it's just a template, rather than a variable definition. See the variable in etc/kayobe/kolla-image-tags.yml.

Just trying it out really, thought it has two benefits:
* Avoid keeping the _tag variables in sync (I almost got the prometheus one wrong)
* Keep the globals.yml file more static. It often gets modified in client repos leading to collisions.

Reply:
It might be useful in yoga, but I don't think we need to worry too much about changing it there and merging it up because it's release-specific and will conflict anyway